go.mod

```diff
@@ -6,20 +6,21 @@ require (
 	github.com/ChainSafe/go-schnorrkel v0.0.0-20200115165343-aa45d48b5ed6
 	github.com/ChainSafe/log15 v1.0.0
 	github.com/OneOfOne/xxhash v1.2.5
+	github.com/allegro/bigcache v1.2.1 // indirect
```

Comment:
any reason the go mods changed?

Reply:
I ran go mod tidy at some point. I can revert if necessary.